cfgov/jinja2/v1/_includes/organisms/info-unit-group-2.html

```diff
@@ -11,6 +11,9 @@
    value:                         Object defined from a StreamField block.
 
    value.format:                  The number and width of columns to output.
+   
+   value.has_top_border:          Boolean for whether or not to add top
+                                  border to info-unit-group
```

Comment:
It might be worth noting here (and in the other templates for the organisms we're adding the option to) that this field is used by `render_block.html` to modify the wrapping `div.block`.

Reply:
Updated in https://github.com/cfpb/cfgov-refresh/pull/3476/commits/082d1b1bd65d8496ff7e23272d60267f81c565d4